.github/workflows/ci.yml

```diff
@@ -2,15 +2,23 @@ name: Keycloak CI
 
 on: [push, pull_request]
 
+env:
+  DEFAULT_JDK_VERSION: 8
+
 jobs:
   build:
-    name: Build
+    name: Build (JDK ${{ matrix.java }})
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [8, 11]
```

Comment:
Do you mean creating another job for building only Quarkus dist? Any specific issue running in both versions considering the main distribution should also support 11 (and we might want in the future to also run tests using v11 too)?

Reply:
@vmuzikar Btw, there is no change to configuration to change the current settings to remove the main dist from the reactor when building using the `distribution` profile. Do you want to add more changes to avoid building main dist when using 11?

Reply:
Ok, let's leave it.